modoboa_installer/ssl.py

```diff
@@ -83,7 +83,7 @@ def install_certbot(self):
                 utils.exec_cmd("add-apt-repository -y ppa:certbot/certbot")
             package.backend.update()
             package.backend.install("certbot")
-        elif name == "Debian":
+        elif name == "Debian" or name == "debian":
```

Comment:
Hi @ReneRumetshofer 

Can you use lower case like 
`
else name.lower() == "debian":
`

Reply:
I've changed the file.
Regards